packages/components/src/components/button/_mixins.scss

```diff
@@ -14,13 +249,32 @@
   @include reset;
   @include type-style('body-short-01');
```

Comment:
So I could be wrong, and this could totally be out of scope for this RFC, but I was looking at it as *variants/modifiers/states* are part of the component, and if we are testing out the validity of a per-component basis it should involve this update and proof out if it can handle outliers like these. I'll leave it for you to decide, I just wanted to point it out for consideration just in case, and so we don't overlook these things as we continue to push forward. 😄

Reply:
@mattrosno if that spec changed I must have missed it. @oliviaflory can you chime in on the `14px`/`16px`?

Reply:
@mattrosno @photodow ah yes, I *did* say we could live with 16px in our group chat. I think that makes sense from a system perspective. We had been considering leaving small button at 14px for instances within other components, like `code snippet` or `data table`, but I do think 16px looks alright there.

You might hate me for this, but could I retract my earlier statement on changing medium button to 48px? If we keep small button at 32px, I think it makes more sense to leave medium at 40px so that we are not providing two "different" variations that look the same visually.

Reply:
> You might hate me for this, but could I retract my earlier statement on changing medium button to 48px? If we keep small button at 32px, I think it makes more sense to leave medium at 40px so that we are not providing two "different" variations that look the same visually.

@oliviaflory even though the field/medium at `40px` button would now be misaligned with the `text-input` which would be `48px` tall?